src/components/LayoutStation/LayoutStation.tsx

```diff
@@ -34,7 +34,7 @@ export function LayoutStation({ ...props }) {
   return (
     <div
       data-testid="layout-station"
-      className={`bg-primary h-screen ${customClass}`}
+      className={`bg-primary h-screen ${customClass} px-20 pt-12 pb-8`}
```

Comment:
This should be BEFORE customClass! CustomClass works to override things. The way you did if we need the CustomClass it wont work anymore.

```suggestion
      className={`bg-primary h-screen px-20 pt-12 pb-8 ${customClass}`}
```